Dockerfile.in

```
@@ -0,0 +1,14 @@
+FROM alpine:3.9.5
+ARG AWS_REGION
+ARG AWS_ACCESS_KEY_ID
+ARG AWS_SECRET_ACCESS_KEY
+ENV HOME /tmp
+
+RUN apk update && \
```

This can be a one-liner that doesn't leave cache files around
```suggestion
RUN apk --no-cache \
```

---

done